chapters/results/sections/non_euclidean/subsections/hyperbolic.tex

```diff
@@ -0,0 +1,29 @@
+\subsection{Hyperbolic space}
```

Comment:
Think we should include the option to change how hyperbolic the game is. We should also show comparisons of this. Also mention that this can be changed during gameplay (something which hyperbolica did not have I think)